base_classes/NXbeam_transfer_matrix_table.nxdl.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+#
+# Copyright (C) 2024-2025 NeXus International Advisory Committee (NIAC)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXbeam_transfer_matrix_table" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <symbols>
+        <doc>
+            Variables used throughout the document, e.g. dimensions or parameters.
+        </doc>
+        <symbol name="N_variables">
+            <doc>
+                Length of the array associated to the data type.
+            </doc>
+        </symbol>
+    </symbols>
+    <doc>
+        Contains datastructures of an experimental optical setup (i.e., multiple
+        transfermatrix tables).   These datastructures are used to relate physical
```

Comment:
```suggestion
        transfer matrix tables).   These data structures are used to relate physical
```
and other run-on compound nouns below

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.

There is an unnecessary duplication of whitespace but as changing this does not require a vote we can still consider the comment resolved and fix this later.